cogs/api.py

```diff
@@ -168,6 +174,17 @@ async def convert(self, ctx: GuildContext, argument: str):
             return user
 
 
+class CreateHelpThreadModal(discord.ui.Modal, title='Create help thread'):
+    thread_name = discord.ui.TextInput(label='Thread title', placeholder='Name for the help thread...', min_length=15, max_length=100)
```

Comment:
```suggestion
    thread_name = discord.ui.TextInput(label='Thread title', placeholder='Name for the help thread...', min_length=20, max_length=100)
```

Should be 20 because of the [title automod](https://github.com/Rapptz/RoboDanny/blob/rewrite/cogs/dpy.py#L313).

Reply:
Good catch, but go back to being chronically offline!

Reply:
My bad.